docs/comparison.md

```diff
@@ -117,7 +118,7 @@ for instance.
 | Commutativity Guarantees                                | ✅                                                                    | 🛑                 | 🛑                                             |
 | Partial Results                                         | ✅                                                                    | ✅                 | 🛑                                             |
 | Safe Partial Results (schema-based)                     | ✅                                                                    | 🛑                 | 🛑                                             |
-| Offline Support                                         | ✅                                                                    | 🛑                 | 🛑                                             |
+| Offline Support                                         | ✅                                                                    | 🟡 `@wora/apollo-offline`| 🟡 `react-relay-offline` `relay-angular`|
```

Comment:
Why is angular mentioned here as well

Reply:
Because the relay-angular library also supports offline

Reply:
Let's remove `relay-angular` from here since we have the other option already, since you already added a Relay option 😅

Reply:
Also, I'd love to wait for the Apollo offline entry and get that removed first. We've spoken to the contributor before (and you of course! 😅) on that other issue, but I've sent this document over to some people at Apollo and would like them to decide whether it should be added here or not

Reply:
You did well :)

I added it because there are third party libraries that enable Apollo to support offline